js/main.js

```diff
@@ -54,17 +55,36 @@ function extractResults(data) {
   return extractedResults;
 }
 
-searchButton.addEventListener('click', (event) => {
-  event.preventDefault();
-  fetchVolumes();
-  form.reset();
-});
+function viewSwap(elementToSwap) {
+  const search = document.querySelector('.search');
+
+  if(elementToSwap === 'searchResults'){
+    search.classList.add('hidden');
+  }
+  // will build on this when clicking find a book, your bookshelf, or adding book to bookshelf
+}
 
-// Issue 2 task
 function renderSearch(resultsToDisplay) {
```

Comment:
if (!resultsToDisplay) {
   //return a "No Books Found" div
} else {
   ...
}

can add this conditional instead to simplify rendering an empty search